package/yast2-network.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-network
-Version:        3.1.171
+Version:        3.2.1
```

Comment:
I had to "correct history" (like dictators do) because the previous patch didn't jump to 3.2.0 as expected.